### ISSUE TITLE
Liquid Monitor does not see Ender IO Fluid Tank (Pressurized or regular)

### ISSUE THREAD

**Issue by sandalle** [McJty/RFTools#1004]
See https://forum.feed-the-beast.com/threads/rftools-liquid-monitor-does-not-see-enderio-fluid-tank.209913/

I also see this on my custom modpack with:
* Minecraft 1.10.2
* Forge Mod Loader (FML) 1.10.2-12.18.3.2185
* RFTools 1.1x-5.82
* EnderCore 1.10.2-0.4.1.65-beta
* Ender IO 1.10.2-3.1.156

Can see an EnderStorage Ender Tank next to it.

**Comment by McJty** [comment]
Did you report this to the EnderIO people? This is probably caused by them not implementing 'null' facing properly for their tank

**Comment by McJty** [comment]
Will be fixed in the next release of EnderIO

**Comment by sandalle** [comment]
Thank you. :) I started here as there was also a thread mentioning it when I searched and Liquid Monitor is the device I want to use. I didn't see a similar thread nor [bug for Ender IO](https://github.com/SleepyTrousers/EnderIO/issues), should I open one to track it? I found one where you talked with them about null facing, but that was for crashing the client, not Liquid Monitor (or similar) working and it was fixed/closed back in August 2016.
